Lib/test/test__interpreters.py

```diff
@@ -555,12 +555,12 @@ class RunStringTests(TestBase):
 
     def setUp(self):
         super().setUp()
-        self.id = _interpreters.create()
+        self._id = _interpreters.create()
```

Comment:
Could you just add a comment about the fact that the `_` must be kept? like adding a comment like `see https://github.com/python/cpython/pull/121704#discussion_r1677409015`

Reply:
Added.